rpc.cc

```diff
@@ -7,14 +7,15 @@ uint16_t client_port = 44445;
 uint16_t server_port = 44444;
 }
 
-RpcChannel::RpcChannel(Pier pier, const std::string& pier_ip)
+RpcChannel::RpcChannel(Pier pier, const std::string& pier_ip, RepStatus& rc)
```

Comment:
It would be better to refactor this code into a separate function. "Connect" , for example, and it will return the status

Reply:
done